packages/react/src/dialog/CloseIcon.tsx

```diff
@@ -0,0 +1,14 @@
+export const CloseIcon = (
+  <svg
+    fill="none"
+    height="16"
+    viewBox="0 0 20 16"
+    width="20"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M14.8436 12.875C14.6561 13.0625 14.3124 13.0625 14.1249 12.875L9.99988 8.71875L5.84363 12.875C5.65613 13.0625 5.31238 13.0625 5.12488 12.875C4.93738 12.6875 4.93738 12.3438 5.12488 12.1562L9.28113 8L5.12488 3.875C4.93738 3.6875 4.93738 3.34375 5.12488 3.15625C5.31238 2.96875 5.65613 2.96875 5.84363 3.15625L9.99988 7.3125L14.1249 3.15625C14.3124 2.96875 14.6561 2.96875 14.8436 3.15625C15.0311 3.34375 15.0311 3.6875 14.8436 3.875L10.6874 8L14.8436 12.1562C15.0311 12.3438 15.0311 12.6875 14.8436 12.875Z"
+      fill="#080736"
```

Comment:
fill needs to be controllable from css
```suggestion
      fill="currentColor"
```